cmake/external_llvm_prebuilt.cmake

```diff
@@ -61,83 +73,163 @@ if (NGRAPH_CPU_ENABLE)
     set(LLVM_INCLUDE_DIR "${source_dir}/include")  # used by other external projects in current scope
     set(LLVM_LIB_DIR "${source_dir}/lib" PARENT_SCOPE)
 
-    set(LLVM_LINK_LIBS
-        clangTooling
-        clangFrontendTool
-        clangFrontend
-        clangDriver
-        clangSerialization
-        clangCodeGen
-        clangParse
-        clangSema
-        clangStaticAnalyzerFrontend
-        clangStaticAnalyzerCheckers
-        clangStaticAnalyzerCore
-        clangAnalysis
-        clangARCMigrate
-        clangRewriteFrontend
-        clangEdit
-        clangAST
-        clangLex
-        clangBasic
-        LLVMLTO
-        LLVMPasses
-        LLVMObjCARCOpts
-        LLVMSymbolize
-        LLVMDebugInfoPDB
-        LLVMDebugInfoDWARF
-        LLVMMIRParser
-        LLVMCoverage
-        LLVMTableGen
-        LLVMDlltoolDriver
-        LLVMOrcJIT
-        LLVMObjectYAML
-        LLVMLibDriver
-        LLVMOption
-        LLVMX86Disassembler
-        LLVMX86AsmParser
-        LLVMX86CodeGen
-        LLVMGlobalISel
-        LLVMSelectionDAG
-        LLVMAsmPrinter
-        LLVMDebugInfoCodeView
-        LLVMDebugInfoMSF
-        LLVMX86Desc
-        LLVMMCDisassembler
-        LLVMX86Info
-        LLVMX86AsmPrinter
-        LLVMX86Utils
-        LLVMMCJIT
-        LLVMLineEditor
-        LLVMInterpreter
-        LLVMExecutionEngine
-        LLVMRuntimeDyld
-        LLVMCodeGen
-        LLVMTarget
-        LLVMCoroutines
-        LLVMipo
-        LLVMInstrumentation
-        LLVMVectorize
-        LLVMScalarOpts
-        LLVMLinker
-        LLVMIRReader
-        LLVMAsmParser
-        LLVMInstCombine
-        LLVMTransformUtils
-        LLVMBitWriter
-        LLVMAnalysis
-        LLVMProfileData
-        LLVMObject
-        LLVMMCParser
-        LLVMMC
-        LLVMBitReader
-        LLVMCore
-        LLVMBinaryFormat
-        LLVMSupport
-        LLVMDemangle
-        tinfo
-        z
-        m
-        PARENT_SCOPE)
-
+    if(APPLE)
```

Comment:
Any way we could have a list of things common to everything augmented by the platform-specific stuff?

Reply:
Sure.
Want to do the same for  cmake/external_llvm.cmake ? I copied the code from that file.

Reply:
There is only one line difference across platforms so the updated code will look much cleaner.